src/ukv/ukv.hpp

```diff
@@ -0,0 +1,100 @@
+#include <cassert>
+
+#include <ukv/cpp/ukv.hpp>
+#include <ukv/cpp/blobs_collection.hpp>
+
+#include "src/core/db.hpp"
+#include "src/core/helper.hpp"
+#include "src/core/types.hpp"
+
+namespace ukv {
+
+namespace fs = ucsb::fs;
+using namespace unum::ukv;
+
+using key_t = ucsb::key_t;
+using keys_spanc_t = ucsb::keys_spanc_t;
+using value_span_t = ucsb::value_span_t;
+using value_spanc_t = ucsb::value_spanc_t;
+using values_span_t = ucsb::values_span_t;
+using values_spanc_t = ucsb::values_spanc_t;
+using value_lengths_spanc_t = ucsb::value_lengths_spanc_t;
+using operation_status_t = ucsb::operation_status_t;
+using operation_result_t = ucsb::operation_result_t;
+using db_hints_t = ucsb::db_hints_t;
+using transaction_t = ucsb::transaction_t;
+
+class ukv_t : public ucsb::db_t {
+    ukv_t(ukv_str_view_t config = nullptr, ) : db_(nullptr), config_(config), arena_(nullptr) {}
```

Comment:
You don't need a comma. And you need separate `arena_`-s if you intend to call this from multiple threads.